receiver/vcenterreceiver/metadata.yaml

```diff
@@ -295,6 +295,18 @@ metrics:
       aggregation_temporality: cumulative
     attributes: [throughput_direction, object_name]
     extended_documentation: As measured over the most recent 20s interval.
+    warnings:
+      if_configured: this metric is replaced by [vcenter.host.network.packet.error.rate] & will be removed starting in release v0.102.0
+  vcenter.host.network.packet.error.rate:
+    enabled: false
+    description: The summation of packet errors transmitted or received on the host network.
```

Comment:
What does summation mean in this context? It reads to me like it's not describing a rate.

Reply:
@djaglowski good point. I'll tweak the descriptions to be a little more obvious that these are rates.

Reply:
@djaglowski should have a better description now